interpretation/src/main/kotlin/com/hartwig/actin/treatment/TreatmentRankingModel.kt

```diff
@@ -60,22 +81,34 @@ class TreatmentRankingModel(private val scoringModel: EvidenceScoringModel) {
         }
 
     private fun treatmentEvidencesWithTargets(actionables: Sequence<Actionable>) =
-        actionables.flatMap {
-            it.evidence.treatmentEvidence.map { t ->
+        actionables.flatMap { actionable ->
+            val directEvidences = actionable.evidence.treatmentEvidence.asSequence().map { treatmentEvidence ->
```

Comment:
Also related to comment on MolecularMatchDetails, I think in the long run it'll be nice to not have to deal with two collections (although I see the defensive value for now). Would be a good follow up task to assess impact of going back to one treatment evidence collection with a flag indicating if its direct/indirect.